### PR TITLE
Issue 1193: update trufflehog command on the challenge 1 hint doc

### LINKED ISSUE #1193: [Docs] Hint on challenge-1 needs an update (trufflehog)
Hi there,

I think the hint on challenge-1 needs an update:

> Scan the files using `trufflehog . | grep password` and the password will be in the output.

```
➜  wrongsecrets git:(master) trufflehog --version
trufflehog 3.59.0
➜  wrongsecrets git:(master) trufflehog .
trufflehog: error: expected command but got ".", try --help
```

### PR DESCRIPTION
**Pull request by za**
### What kind of changes does this PR include?

-   [ ] Fixes or refactors
-   [ ] A new challenge
-   [x] Additional documentation
-   [ ] Something else

#### Description

because with previous command it's not showing the result. Trufflehog needs to scan the filesystem and use generic detector from the example.

Note: it's suggested to use the latest Trufflehog version. When this
commit was made, the latest version is: 3.63.10

### Relations

Closes #1193 

### Checklist:

-   [x] All the contributions made are solely the work of me and my co-authors

### CODE REVIEW

File: src/main/resources/explanations/challenge1_hint.adoc [hunk 1]
```diff
@@ -21,4 +21,6 @@ You can solve this challenge by the following steps:
 4. You can scan the repository with https://github.com/trufflesecurity/trufflehog[*Trufflehog*].
 - Clone the repo with `git clone https://github.com/OWASP/wrongsecrets`.
 - Follow the instructions https://github.com/trufflesecurity/trufflehog[here] to install Trufflehog.
-- Scan the files using `trufflehog . | grep password` and the password will be in the output.
+- How to download trufflehog https://github.com/trufflesecurity/trufflehog/blob/4afc224c635d10e732119f715f93788af1502ce4/examples/README.md[generic detector] file
```
Comment (commjoen) [inline src/main/resources/explanations/challenge1_hint.adoc:24]:
```suggestion
- Follow these instructions to download the https://github.com/trufflesecurity/trufflehog/blob/4afc224c635d10e732119f715f93788af1502ce4/examples/README.md[generic detector] file
```

File: src/main/resources/explanations/challenge1_hint.adoc [hunk 2]
```diff
@@ -21,4 +21,6 @@ You can solve this challenge by the following steps:
...
+- Download trufflehog generic detector `wget https://raw.githubusercontent.com/trufflesecurity/trufflehog/main/examples/generic.yml`.
+- Scan the files using `trufflehog filesystem --config=$PWD/generic.yml . | grep password` and the password will be in the output.
```
Comment (commjoen) [inline src/main/resources/explanations/challenge1_hint.adoc:26]:
```suggestion
- Scan the files using `trufflehog filesystem --config=$PWD/generic.yml . | grep password` and the password will be in the output.

5. Alternativel you can use trufflehog2:
- pip3 install trufflehog
- Scan the files using `trufflehog . | grep password` and the password will be in the output.
```

**Review by commjoen — APPROVED** [review]
Thank you @za !

**Comment by za** [comment]
Thanks for merging it @commjoen ! 🥳

**Pull request merged**